package.json

```diff
@@ -208,6 +208,7 @@
     "prettier-bytes": "^1.0.4",
     "qr-image": "^3.2.0",
     "react-json-view-lite": "^0.9.5",
+    "react-lottie-player": "^2.1.0",
```

Comment:
prefer to pin exact version. any way to do this without adding a new dependency?

Reply:
this also adds two new subdependencies, lottie-web and rfdc. if we are going to add this new dependency could we make sure the CSP for the webUI page that it's loaded in disallows `unsafe-eval`? it might already be the default. https://github.com/brave/reviews/issues/1627#issuecomment-2136120275

Reply:
See comment in `vpn-shields-connecting.tsx` about avoiding the new dependency.